source/utilities.tex

```diff
@@ -19935,6 +19935,8 @@
 the first code points in its extended grapheme clusters
 as defined by
 Unicode Standard Annex \#29 Unicode Text Segmentation.
+
+\begin{example}
```

Comment:
I believe this is the definition of "estimated width", not an example.

The "as defined by" is referring only to "extended grapheme clusters" (which are defined by Annex \#29), not to "estimated width" (which is not defined by Annex \#29).

Reply:
@zygoloid, a later commit corrects this.

Reply:
? No, the example markers were present at the tip of the branch. I've removed them now.